packages/api/endpoints/granules.js

```diff
@@ -95,9 +95,52 @@ const create = async (req, res) => {
   return res.send({ message: `Successfully wrote granule with Granule Id: ${granule.granuleId}` });
 };
 
+/**
+ * Update existing granule
+ *
+ * @param {Object} req - express request object
+ * @param {Object} res - express response object
+ * @returns {Promise<Object>} promise of an express response object.
+ */
+const update = async (req, res) => {
+  const {
+    granuleModel = new models.Granule(),
+    knex = await getKnexClient(),
+  } = req.testContext || {};
+  const body = req.body || {};
+
+  let existingGranule;
+  try {
+    existingGranule = await granuleModel.get({ granuleId: body.granuleId });
+  } catch (error) {
+    if (error instanceof RecordDoesNotExist) {
+      return res.boom.badRequest(`No granule found to update for ${JSON.stringify(body)}`);
```

Comment:
This may be a result of debugging I was doing.  But definitely makes sense to only return the body.granuleId since that's all that was used to determine it doesn't have a record.

Reply:
61d890eaa6